CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - pipe
  - splitAt/asyncSplitAt
  - multiPartition/asyncMultiPartition
+ - mergeAll, asyncMergeAll
```

Comment:
This is a future feature.
But I can fix it for you @conartist6 
Is this ready to be merged ?

Reply:
Thanks that would be great. I'm working on the `interleaveGenerator` now. I do believe this is ready to go.